Source/WebKit/ChangeLog

```diff
@@ -1,3 +1,14 @@
+2022-05-13  Michael Catanzaro  <mcatanzaro@redhat.com>
+
+        [GTK] Warning in WebKitDOMDocumentGtk.cpp with GCC 12
+        https://bugs.webkit.org/show_bug.cgi?id=239353
+
+        Reviewed by NOBODY (OOPS!).
+
```

Comment:
Do we know why this false-positive is happening? If we know that, let's put description here.

Reply:
Don't know....